### ISSUE TITLE
Black and red product on white background

### ISSUE THREAD

**Issue by johntunnel** [danielgatis/rembg#138]
Hi
I have a problem with rembg cutting out black and red product on white background. Below is original and cut out image with default setting and another cut out with some masking setting.
![press29](https://user-images.githubusercontent.com/35736520/133194302-b19c2839-18d7-4b05-ad70-666b8553991e.JPG)
![press29B](https://user-images.githubusercontent.com/35736520/133194372-ce1289c1-3a8f-4472-83b7-c67180a52a59.JPG)
![press29D](https://user-images.githubusercontent.com/35736520/133194380-6fb49e1f-fb00-497c-aca3-83622d36b722.PNG)
Is there anything I can do to improve it?
I have a sequence of this product images on turntable and some have better results, some not. Is it suitable to do own model? How to do own model? I have sequence of similar images which may be used for modeling. I could also do a photo of only white background picture without product if this would help. Such white background would be 98% similar as the background in the problematic picture (not pixel perfect due to possible misalignment, maybe a little different lighting). 
Any suggestions would be appreciated.

**Comment by johntunnel** [comment]
I was looking at https://pytorch.org/tutorials/beginner/basics/intro.html# to understand better how to make a new model and now I see this modeling is done based on known objects (pictures of same object pretrained). Since my object is "strange" shape it does not fit into any pretrained category and thus is not so well cut out. In fact probably rembg is much better suited to cut out known objects in the image than for other products which have strange shapes (like above one). Training for shape recognition in case of above product is not suitable, because this product is very rare. I would need some other tool which cuts out by looking at colours only, not at shapes. 
Is my thinking correct?

**Comment by danielgatis** [comment]
https://github.com/danielgatis/rembg/issues/129#issuecomment-897081037
